src/main/java/HomeWorks/HW2Servlets/ServletFive.java

```diff
@@ -0,0 +1,37 @@
+package HomeWorks.HW2Servlets;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class ServletFive extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```

Comment:
If we don't need method we don't ovveride method, it is not necessary

Reply:
fixed